drivers/serial/uart_rpi_pico.c

```diff
@@ -40,6 +52,9 @@ static void uart_rpi_poll_out(const struct device *dev, unsigned char c)
 static int uart_rpi_init(const struct device *dev)
 {
 	const struct uart_rpi_config *config = dev->config;
+	uart_inst_t * const uart_inst = config->uart_dev;
+	uart_hw_t * const uart_hw = config->uart_regs;
+	struct uart_rpi_data * const data = dev->data;
```

Comment:
This also needs to be in the previous commit.

Reply:
Done!